test/microbench/org/apache/cassandra/test/microbench/CacheLoaderBench.java

```diff
@@ -97,7 +97,7 @@ public void setup() throws Throwable
                 RowUpdateBuilder rowBuilder = new RowUpdateBuilder(cfs.metadata(), System.currentTimeMillis() + random.nextInt(), "key");
                 rowBuilder.add(colDef, "val1");
                 rowBuilder.build().apply();
-                cfs.forceBlockingFlush();
+                cfs.forceBlockingFlush(ColumnFamilyStore.FlushReason.UNIT_TESTS);
```

Comment:
Not super important, but I notice we're using `UNIT_TESTS` in some jmh tests and `USER_FORCED` in others?